src/modules/pageDetails.scss

```diff
@@ -36,7 +36,7 @@ $divider-color: rgba(255, 255, 255, .25);
 
 @media only screen and (min-width: 768px) {
     .ui.page--details {
-        &.page--details-bleed {
+        &.details-bleed {
             margin: 0 -22px;
         }
         .ui.info-bar {
```

Comment:
Let's strip some of the common css out of this file and make a `details.scss` file so that it only applies to the two different components.

Reply:
For this, Just need a little more clarification. Striping out  common css from `pageDetails.scss` & `drawerDetails.scss` and placing it into `details.scss`. Here's what I am thinking, need to add new common class which would work for both page and drawer component like below
`className={`${moduleType}--details-columns-container details-columns-container`}`
Here, `details-columns-container` is new common class. I hope you are getting my idea. Am I thinking correctly please suggest?

Reply:
Fixed